ir/visitor.h

```diff
@@ -37,7 +37,7 @@ struct Visitor_Context {
     int                         depth;
 };
 
-class Visitor : public ICastable {
+class Visitor  {
```

Comment:
This was introduced in #3542. I do not think visitors were ever castable? Instead trigger, was castable.